src/string/to-upper.ts

```diff
@@ -1,7 +1,23 @@
 import { Type, Kind } from '..'
 
+/**
+ * `String._$toUpper` is a type-level function that converts a string to uppercase.
+ *
+ * @template S - The string to convert to uppercase.
+ *
+ * @example
+ * type T0 = String._$toUpper<'foo'> // 'FOO'
+ * type T1 = String._$toUpper<'bar'> // 'BAR'
+ */
 export type _$toUpper<S extends string> = Uppercase<S>
 
+/**
+ * `String.ToUpper` is a type-level function that converts a string to uppercase.
+ *
```

Comment:
`@template` seems to be missing
```suggestion
 * @template S - The string to convert to uppercase.
```